src/views/Package/PackageDocs.tsx

```diff
@@ -114,6 +241,27 @@ export const PackageDocs: FunctionComponent = () => {
           </Route>
         </Switch>
       </Box>
+      <Flex
+        alignSelf="stretch"
+        borderLeft="1px solid"
+        borderLeftColor="borderColor"
+        direction="column"
+        display={{ base: "none", lg: "flex" }}
+        maxHeight={`calc(100vh - ${TOP_OFFSET})`}
+        overflow="hidden auto"
+        pl={6}
+        position="sticky"
+        pr={4}
+        top={TOP_OFFSET}
+      >
```

Comment:
This flex container has a lot in common with the main nav container. We can create a common component for the two of them either in a separate file:

```tsx
// Feel free to name it something catchier
export const StickyNavContainer: typeof Flex = (props) => (
  <Flex
    as="nav" // to be more semantic
    alignSelf="stretch"
    direction="column"
    display={{ base: "none", lg: "flex" }}
    maxHeight={`calc(100vh - ${TOP_OFFSET})`}
    overflow="hidden auto"
    position="sticky"
    {...props}
  />
);

```

Reply:
Updated